### PR TITLE
[fix bug 1250433] Update /firefox/accounts/ page conditional states

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Updates `/firefox/accounts` page with conditional states for:

- [x] Firefox desktop (not-signed in users).
- [x] Firefox desktop (already signed in users).
- [x] Firefox mobile users (iOS & Android).
- [x] Legacy Firefox users (pre-FF 39).
- [x] Non-Firefox users.

This PR makes both the `/firefox/accounts` page and homepage promo tile ready for localization. Also cleans up the old waffle/GA naming conventions that we're previously 'mau2account'.

## Bugzilla
https://bugzilla.mozilla.org/show_bug.cgi?id=1250433

## Testing
* Please follow the bedrock documentation for[ testing FxA iFrame](http://bedrock.readthedocs.org/en/latest/firefox-accounts.html)
* Also make sure you have [UITour enabled](http://bedrock.readthedocs.org/en/latest/uitour.html#local-development).
- Branch is also up on demo2: https://www-demo2.allizom.org/en-US/firefox/accounts/

## Checklist
- [x] Requires l10n changes.
- [ ] Related functional & integration tests passing - none exist for this page (yet).

### CODE REVIEW

**Comment by alexgibson** [comment]
cc @flodolo - here be final strings. Will try and get this on a demo server too.

**Comment by alexgibson** [comment]
This is now also up on demo2: https://www-demo2.allizom.org/en-US/firefox/accounts/

File: media/js/firefox/accounts.js [hunk 1]
```diff
@@ -124,27 +122,81 @@
...
+        $main.addClass('single-column');
+        $('.mobile').show();
+        $('.firefox-ios').css('display', 'inline-block');
+    }
+
+    // Firefox Android users get SUMO link
+    function showFirefoxAndroid() {
+        $main.addClass('single-column');
+        $('.mobile').show();
+        $('.firefox-android').css('display', 'inline-block');
```
Comment (jpetto) [inline media/js/firefox/accounts.js]: Not a blocker for this PR, but wondering if we should begin enforcing `display` changes with classes? Could be a nice technical debt task.

**Comment by jpetto** [comment]
r+ on the code, but I believe we're waiting on Gareth to review GA prior to merge.

File: media/js/firefox/accounts.js (hunk 1 above)
Comment (alexgibson) [inline media/js/firefox/accounts.js]: A worthwhile suggestion :+1:

**Comment by alexgibson** [comment]
Thanks for the review @jpetto updated - will likely have one more commit to make to add the new GA events before we're good to merge.

**Comment by jpetto** [comment]
Double-checked conditional states with the new CSS and still looking good. :blue_car:

**Comment by alexgibson** [comment]
@jpetto - I went and added some made up GA events for now for each state. We can sync up with Gareth later when he has more time to look at this. Removing the blocked/do-not-merge labels :+1:

**Comment by jpetto** [comment]
Added GA events look sane to me. I'd say we're good for now.

**Pull request merged**